src/cpu/x64/jit_brgemm_post_ops.cpp

```diff
@@ -52,21 +50,19 @@ dnnl::impl::cpu::x64::jit_brgemm_kernel_diff_bias_t<Vmm>::
     , brg_(abrg)
     , reduce_kind_(bgmmc.reduce_kind)
     // MatMul `src`.
-    , ddst_dt_(bgmmc.src_dt)
+    // This kernel must be called after the copy A routine because it assumes
+    // that fp16 data has already been upconverted to f32.
```

Comment:
This comment was not about `ddst_dt_` argument so we should leave it in the ctor body.

Reply:
Restored, thanks.